### DISCUSSION TITLE
Vertex Paint Not Working

### DISCUSSION THREAD

**Discussion by christypious** [naverz/zepeto-studio-kor#717] [5. Q&A]
Hello! Please help. 

I have built a custom skin for the character and the parts of the mask object that are hidden, I have painted them black. When I import to unity and convert to a zepeto object, it is supposed to be hidden and yet it is visible. 

Thank you

**Comment by Zepeto-TechEvangelist** [comment]
Hello, This is the ZEPETO World Development Community.
Please inquire about the item issue through the customer center.
https://support.zepeto.me/hc/en-us

Thank you.
